addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/TechPanel.java

```diff
@@ -205,13 +223,15 @@ protected JXTable getTechTable() {
             techTable.setRowSelectionAllowed(true);
             techTable.setAutoCreateRowSorter(true);
             techTable.setColumnControlVisible(true);
+            TableCellRenderer renderer = new DefaultTableRenderer(new MappedValue(
+                    StringValues.EMPTY, IconValues.ICON), JLabel.CENTER);
+            techTable.setDefaultRenderer(ImageIcon.class, renderer);
```

Comment:
I also tried this as:
```java
            TableColumn iconColumn = techTable.getColumn(Constant.messages.getString("wappalyzer.table.header.icon"));
            iconColumn.setCellRenderer(renderer);
```

Neither seems to work. It still exported or copy/pasted as the icon path (when there was an icon).

Reply:
I'd suggest removing the icon from the table model, that belongs to the UI. The first column should return the `Application` (which `toString()` returns the name) then we can show the name and the icon in the same column and export/copy just the name.
The renderer would be:
```Java
TableCellRenderer renderer =
        new DefaultTableRenderer(
                new MappedValue(
                        StringValues.TO_STRING,
                        item -> {
                            if (item == null) {
                                return null;
                            }
                            Icon icon = ((Application) item).getIcon();
                            return icon != null ? icon : TRANSPARENT_ICON;
                        }),
                JLabel.LEADING);
techTable.setDefaultRenderer(Application.class, renderer);
```
with `TRANSPARENT_ICON` as, e.g.:
```Java
private static final Icon TRANSPARENT_ICON =
    new Icon() {

        @Override
        public void paintIcon(Component c, Graphics g, int x, int y) {
            // Nothing to do.
        }

        @Override
        public int getIconWidth() {
            return 32;
        }

        @Override
        public int getIconHeight() {
            return 32;
        }
    };
```